internal/app/decision_maker.go

```diff
@@ -149,37 +153,46 @@ var releaseNameExtractor = regexp.MustCompile(`sh\.helm\.release\.v\d+\.`)
 // getHelmsmanReleases returns a map of all releases that are labeled with "MANAGED-BY=HELMSMAN"
 // The releases are categorized by the namespaces in which they are deployed
 // The returned map format is: map[<namespace>:map[<helmRelease>:true]]
-func (cs currentState) getHelmsmanReleases() map[string]map[string]bool {
+func (cs *currentState) getHelmsmanReleases(s *state) map[string]map[string]bool {
 	var lines []string
+	const outputFmt = "custom-columns=NAME:.metadata.name,NS:.metadata.namespace,CTX:.metadata.labels.HELMSMAN_CONTEXT"
```

Comment:
these changes should fix #355 and make things faster since only one `kubectl` command is issued to get all releases.